services/datalad/tests/dataset_fixtures.py

```diff
@@ -98,6 +98,7 @@ def new_dataset(datalad_store):
     with open(changes_path, 'w') as f:
         f.write(CHANGES)
     ds.save(changes_path)
+    ds.close()
     return ds
```

Comment:
Is the dataset still usable once closed?

Reply:
Yes, it just forces DataLad to reopen the repo from the filesystem on the next access.